doc/modules/ROOT/pages/common-datasets.adoc

```diff
@@ -0,0 +1,84 @@
+= Common datasets
+
+
+For convenience, the library is shipped with a few common datasets.
+We can easily load it into GDS and get a graph object representing the dataset.
+
+Every pre-canned dataset comes with a loader method that takes two optional parameters:
+`graph_name` which assigns a graph name,
+`undirected` which takes a boolean and will load the graph as undirected if set to true.
+
+If a graph is loaded as `undirected = True`, then it will have twice the number of relationships compared to its directed version.
+The default value for `undirected` varies for each dataset.
+
+For example:
+[source, python]
+----
+G = gds.graph.load_cora()
+
+assert G.node_count() == 2708
+assert G.node_labels() == ["Paper"]
+----
+
+
+== Datasets
+
+=== Cora
+A well known citation network introduced by
+https://doi.org/10.1023/A:1009953814988[Automating the Construction of Internet Portals with Machine Learning]
+and used in many node classification or link prediction https://paperswithcode.com/dataset/cora[publications].
+
+The default is to load Cora as `undirected = False`.
+
+.The loaded cora graph object
+[opts="header",cols="3m,3m", role="no-break"]
+|===
+| Name                    | Value
+| name                    | cora
+| node_count              | 2708
+| relationship_count      | 5429
+| node_labels             | ['Paper']
+| relationship_types      | ['CITES']
+| node_properties         | Paper: [features, subject]
```

Comment:
Yeah it could be useful, but I think it might bloat the result as most of the times the type should be clear. Also feels a bit like java-style map. Will leave this for now.

Reply:
All right